trajopt/include/trajopt/problem_description.hpp

```diff
@@ -50,6 +51,14 @@ class TRAJOPT_API TrajOptProb : public sco::OptProb
   TrajOptProb();
   TrajOptProb(int n_steps, const ProblemConstructionInfo& pci);
   virtual ~TrajOptProb() = default;
+  sco::VarVector GetJointVarRow(int i)
```

Comment:
I recommend using the `rblock` function in BasicArray then change this method to GetVarRow(int i, int start_col, int num_col).

Reply:
This has been changed.